assets/kube-apiserver/config.yaml

```diff
@@ -17,120 +17,195 @@ admission:
         restrictedCIDRs:
         - {{ .PodCIDR }}
         - {{ .ServiceCIDR }}
-aggregatorConfig:
-  proxyClientInfo:
-    certFile: "/etc/kubernetes/secret/proxy-client.crt"
-    keyFile: "/etc/kubernetes/secret/proxy-client.key"
 apiServerArguments:
+  advertise-address:
+  - "{{ .ExternalAPIIPAddress }}"
+  allow-privileged:
```

Comment:
I was struggling to find the previous setting for `allow-privileged` along with the following: `anonymous-auth`, `authorization-mode`, `enable-admission-plugins`, `enable-logs-handler`, `enable-swagger-ui`, `endpoint-reconciler-type`, `etcd-prefix`, `event-ttl`, `goaway-chance`, `insecure-port`, `kubelet-https`, `kubelet-read-only-port`, `kubernetes-service-node-port`, `max-mutating-requests-inflight`, and `service-account-lookup`.  Are these the previous defaults?